graviti/manager/commit.py

```diff
@@ -162,7 +163,17 @@ def _getter(offset: int, limit: int) -> Dict[str, Any]:
                 limit=limit,
             )
 
-        total_count = _getter(0, 1)["total_count"]
+        count_criteria = deepcopy(criteria)
+        count_criteria["select"] = [{"count": {"$count": ["$.*"]}}]
```

Comment:
do we need to set limit = 1 here?

Reply:
Dont't need limit here, just the count will be returned.